specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/EnterpriseKnowledgeGraphSwagger.json

```diff
@@ -0,0 +1,2001 @@
+            }
+          },
+          "triples": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "object"
+            }
+          }
+        }
+      },
+      "ISet": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "values": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/INode"
+            }
+          },
+          "variables": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "PatternItem": {
+        "type": "object",
+        "properties": {
+          "bindingContext": {
+            "$ref": "#/definitions/SparqlResultBinder"
+          },
+          "rigorousEvaluation": {
+            "type": "boolean"
+          },
+          "variableName": {
+            "type": "string"
+          },
+          "repeated": {
+            "type": "boolean"
+          }
+        }
+      },
+      "INamespaceMapper": {
+        "type": "object",
+        "properties": {
+          "prefixes": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "INode": {
+        "type": "object",
+        "properties": {
+          "nodeType": {
+            "format": "int32",
+            "description": "\r\n0=Blank; \r\n1=Uri; \r\n2=Literal; \r\n3=GraphLiteral; \r\n4=Variable; \r\n",
+            "enum": [
+              0,
+              1,
+              2,
+              3,
+              4
+            ],
+            "type": "integer"
+          },
+          "graph": {
+            "$ref": "#/definitions/IGraph"
+          },
+          "graphUri": {
+            "type": "string"
+          }
+        }
+      },
+      "SearchOptions": {
+        "description": "Search options specified in Entity Document request body",
+        "type": "object",
+        "properties": {
+          "entities": {
+            "uniqueItems": false,
+            "description": "Gets or sets a list of entity ids for retrieving from Entity Document store",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "SparqlQueryResponse": {
+        "type": "object",
+        "description": "Query Response class for graph engine service",
+        "properties": {
+          "sparqlResponse": {
+            "$ref": "#/definitions/SparqlQueryDbResponse"
+          },
+          "metadata": {
+            "type": "object",
+            "additionalProperties": {
+              "type": "object"
+            }
+          }
+        }
+      },
+      "SparqlQueryDbResponse": {
```

Comment:
I would suggest naming the type in a way that makes it clear that this is a [SPARQL response object](https://www.w3.org/TR/2013/REC-sparql11-results-json-20130321/#json-result-object) and add a description that indicates that this is a standard format, not something that is unique to this service.